### ISSUE TITLE
aehttp_spec_SUITE fails

### ISSUE THREAD

**Issue by gorbak25** [aeternity/aeternity#3334]
aehttp_spec_SUITE fails with 400 Bad Request

**Comment by velzevur** [comment]
Error message is
```
%%% aehttp_spec_SUITE: ..

----------------------------------------------------
2020-08-24 14:40:29.247
Connection problem: {ok,{{"HTTP/1.1",400,"Bad Request"},
                         [{"connection","keep-alive"},
                          {"date","Mon, 24 Aug 2020 14:40:29 GMT"},
                          {"server","Jetty(9.4.18.v20190429)"},
                          {"content-length","76"},
                          {"content-type","application/json"},
                          {"access-control-allow-origin","*"},
                          {"access-control-allow-methods",
                           "GET, POST, DELETE, PUT"},
                          {"access-control-allow-headers",
                           "Content-Type, api_key, Authorization"},
                          {"access-control-expose-headers",
                           "Content-Disposition"}],
                         "{\"code\":400,\"message\":\"Input error: missing required query parameter `url`\"}"}}
```

**Comment by velzevur** [comment]
I see this is an reincarnation of #3324

**Comment by velzevur** [comment]
This seems to have been solved on recent master by #3325 , the error no longer appears on ~HEAD
